### PR TITLE
:bug: Minor Postives_momentum init Fix

### LINKED ISSUE #726: TypeError: '>' not supported between instances of 'NoneType' and 'int'
horovodrun -np 1 -H localhost:1 ../../pyenv/bin/python3 main.py --mode=train_and_eval \
    --training_file_pattern=tfrecord/demo_train2/*.tfrecord \
    --validation_file_pattern=tfrecord/demo_test2/*.tfrecord \
    --model_name=efficientdet-d0 \
    --model_dir=efficientdet-d0-finetune \
    --ckpt=efficientdet-d0 \
    --train_batch_size=2 \
    --eval_batch_size=2 --eval_samples=500 \
    --num_examples_per_epoch=5000 --num_epochs=20 \
    --hparams=aicv_config.yaml \
    --strategy=horovod

image

[1,0]<stderr>:Traceback (most recent call last):                                                                                                                                                                          
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap                                                                             
[1,0]<stderr>:    self.run()                                                                                                                                                                                              
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/multiprocessing/process.py", line 93, in run                                                                                     
[1,0]<stderr>:    self._target(*self._args, **self._kwargs)                                                                                                                                                               
[1,0]<stderr>:  File "main.py", line 361, in run_train_and_eval                                                                                                                                                           
[1,0]<stderr>:    _train(e * FLAGS.num_examples_per_epoch // FLAGS.train_batch_size)                                                                                                                                      
[1,0]<stderr>:  File "main.py", line 277, in _train                                                                                                                                                                       
[1,0]<stderr>:    max_steps=steps)                                                                                                                                                                                        
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 3089, in train                                   
[1,0]<stderr>:    rendezvous.raise_errors()                                                                                                                                                                               
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/error_handling.py", line 150, in raise_errors                            
[1,0]<stderr>:    six.reraise(typ, value, traceback)                                                                                                                                                                      
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/six.py", line 703, in reraise                                                                                      
[1,0]<stderr>:    raise value                                                                                                                                                                                             
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 3084, in train                                   
[1,0]<stderr>:    saving_listeners=saving_listeners)                                                                                                                                                                      
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 349, in train                                            
[1,0]<stderr>:    loss = self._train_model(input_fn, hooks, saving_listeners)                                                                                                                                             
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 1175, in _train_model                                    
[1,0]<stderr>:    return self._train_model_default(input_fn, hooks, saving_listeners)                                                                                                                                     
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 1204, in _train_model_default                            
[1,0]<stderr>:    self.config)                                                                                                                                                                                            
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 2921, in _call_model_fn                          
[1,0]<stderr>:    config)                                                                                                                                                                                                 
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/estimator.py", line 1163, in _call_model_fn                                  
[1,0]<stderr>:    model_fn_results = self._model_fn(features=features, **kwargs)                                                                                                                                          
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 3179, in _model_fn                               
[1,0]<stderr>:    features, labels, is_export_mode=is_export_mode)                                                                                                                                                        
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 1700, in call_without_tpu                        
[1,0]<stderr>:    return self._call_model_fn(features, labels, is_export_mode=is_export_mode)                                                                                                                             
[1,0]<stderr>:  File "/root/efficientdet/pyenv/lib/python3.6/site-packages/tensorflow_estimator/python/estimator/tpu/tpu_estimator.py", line 2031, in _call_model_fn                          
[1,0]<stderr>:    estimator_spec = self._model_fn(features=features, **kwargs)                                                                                                                                            
[1,0]<stderr>:  File "/root/efficientdet/automl/efficientdet/det_model_fn.py", line 638, in efficientdet_model_fn                                                                             
[1,0]<stderr>:    variable_filter_fn=variable_filter_fn)                                                                                                                                                                  
[1,0]<stderr>:  File "/root/efficientdet/automl/efficientdet/det_model_fn.py", line 385, in _model_fn                                                                                         
[1,0]<stderr>:    cls_outputs, box_outputs, labels, params)                                                                                                                                                               
[1,0]<stderr>:  File "/root/efficientdet/automl/efficientdet/det_model_fn.py", line 216, in detection_loss                                                                                    
[1,0]<stderr>:    if params.get('positives_momentum', 0) > 0:                                                                                                                                                             
[1,0]<stderr>:**TypeError: '>' not supported between instances of 'NoneType' and 'int'**

### PR DESCRIPTION
**Pull request by kartik4949**
Fixes #726

### CODE REVIEW

**Comment by LaurensHagendoorn** [comment]
I think maybe a better fix would be to change lines:
https://github.com/google/automl/blob/07f10f23c95753b23ecb482097aabdaffe8b6741/efficientdet/det_model_fn.py#L228
and 
https://github.com/google/automl/blob/07f10f23c95753b23ecb482097aabdaffe8b6741/efficientdet/det_model_fn.py#L229
to 
`params.get('positives_momentum', 0)`

?

**Comment by kartik4949** [comment]
> I think maybe a better fix would be to change lines:
> https://github.com/google/automl/blob/07f10f23c95753b23ecb482097aabdaffe8b6741/efficientdet/det_model_fn.py#L228
> 
> 
> and
> https://github.com/google/automl/blob/07f10f23c95753b23ecb482097aabdaffe8b6741/efficientdet/det_model_fn.py#L229
> 
> 
> to
> `params.get('positives_momentum', 0)`
> ?

Problem was on line if params.get('positives_momentum', 0) > 0:

**Comment by LaurensHagendoorn** [comment]
> > I think maybe a better fix would be to change lines:
> > https://github.com/google/automl/blob/07f10f23c95753b23ecb482097aabdaffe8b6741/efficientdet/det_model_fn.py#L228
> > 
> > and
> > https://github.com/google/automl/blob/07f10f23c95753b23ecb482097aabdaffe8b6741/efficientdet/det_model_fn.py#L229
> > 
> > to
> > `params.get('positives_momentum', 0)`
> > ?
> 
> Problem was on line if params.get('positives_momentum', 0) > 0:
> with error TypeError: '>' not supported between instances of 'NoneType' and 'int'

Oh I see, I guess I also assumed get(None) would resolve to the alternative. nevermind
(in that case the get could be removed all together?)

File: efficientdet/hparams_config.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def default_detection_configs():
   # A temporary flag to switch between legacy and keras models.
   h.use_keras_model = True
   h.dataset_type = None
-  h.positives_momentum = None
+  h.positives_momentum = 0
```
Comment (kartik4949) [inline efficientdet/hparams_config.py:279]: @mingxingtan i think 1 will be the right value ,right?

**Pull request merged**